package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "browser-file-system",
+  "description": "A in-memory file system for use in-browser",
+  "scripts": {
+    "build": "tsc",
+    "test": "node ./node_modules/mocha/bin/mocha"
+  },
+  "dependencies": {},
+  "devDependencies": {
```

Comment:
It would be good to add typescript dependency here instead of using the global one.